Services/LTI/classes/Screen/LtiViewLayoutProvider.php

```diff
@@ -18,75 +29,173 @@
 class LtiViewLayoutProvider extends AbstractModificationProvider implements ModificationProvider
 {
 
+    const GS_EXIT_LTI = 'lti_exit_mode';
+
+    protected function isLTIMode(): bool
+    {
+        return $this->dic["lti"]->isActive();
+    }
+
+    protected function isLTIExitMode(CalledContexts $screen_context_stack): bool
+    {
+        $data_collection = $screen_context_stack->current()->getAdditionalData();
+        return $data_collection->is(self::GS_EXIT_LTI, true);
+    }
+
+    public function isInterestedInContexts() : ContextCollection
+    {
+        return $this->context_collection->lti();
+    }
+
     /**
      * @inheritDoc
      */
-     
-    public function isInterestedInContexts() : ContextCollection
+    public function getPageBuilderDecorator(CalledContexts $screen_context_stack) : ?PageBuilderModification
     {
-        return $this->context_collection->lti();
+        if(! $this->isLTIMode()) {
+            return null;
+        }
+
+        $this->globalScreen()->layout()->meta()->addCss('./Services/LTI/templates/default/lti.css');
```

Comment:
as mentioned in the last PR I can't say if we still should use specific CSS (I understand the usecase). maybe @Amstutz can comment that.

Reply:
The CSS is minimal as I see

Reply:
As I learned, the LTI-standard asks for the possibility to relate some CSS.
It seemed like a minimal invasion to add the lti.css at this point, too - it re-labels the exit button, which is a compromise to a complete mainbar-template in the former PR.